examples/with-one-login-per-subdomain/.env

```diff
@@ -0,0 +1 @@
+HOST='a.example.com'
```

Comment:
Is this needed?

Reply:
Added that so when the app is started it does not default localhost:3000, i wanted it to default to a.example.com when you start up the app.

Reply:
Oww cool!